src/constants.tsx

```diff
@@ -8,6 +8,12 @@ export const networks: Record<string, string> = {
   //governance: "https://ait2.aptosdev.com/", // TODO - use the proposals url
 };
 
+export const walletNetworkMap: Record<string, string> = {
```

Comment:
Can we implement this as a function that takes in the aforementioned enum of wallet networks and returns an enum of valid explorer networks (i.e. `keyof typeof networks` - you will need to remove the `Record<string, string>` cast from `networks` for this to work - you should remove it from `walletNetworkMap` as well), with assertNever in the default case for exhaustiveness checking? This way if we ever add more wallet networks we will be required to update the mapping. This would replace the`getWalletNetworkFromNetworkMap` function later on in the PR.